packages/atomic/tailwind.config.js

```diff
@@ -1,8 +1,14 @@
+const isDevWatch =
+  process.argv &&
+  process.argv.indexOf('--dev') > -1 &&
+  process.argv.indexOf('--watch') > -1;
+
```

Comment:
Why is the `watch` flag important to consider? And I think `process.argv` always a defined array, removing the need for the first condition?

Could the check be simplified to just:
```
const isDev = process.argv.indexOf('--dev') > -1;
// or
const isDev = process.argv.includes('--dev');
```